docs/MorganeyIndex.md

```diff
@@ -0,0 +1,32 @@
+# Morganey Index
+
+Morganey index is a file called `morganey-index` that is located at
+the root of Morganey module container. Morganey module container is a
+place that is discoverable via JVM classpath mechanism (jar, folder,
+etc) and contains Morganey modules. For example, here is the structure
```

Comment:
Also let's add a clarification here: "Morganey runtime can load multiple modules at once, and each of them should contain its own Index file".